lisa/tools/kdump.py

```diff
@@ -217,6 +221,16 @@ def _install(self) -> bool:
         raise NotImplementedError()
 
     def check_required_kernel_config(self) -> None:
+        kexec_config_present = False
+        for config in self.kexec_kernel_configs:
+            if self.node.tools[KernelConfig].is_built_in(config):
+                kexec_config_present = True
+                break
+        if not kexec_config_present:
+            raise LisaException(
+                "The kernel config CONFIG_KEXEC or CONFIG_KEXEC_FILE is not set. Kdump is"
+                "not supported."
+            )
```

Comment:
if not kexec_config_present:
            raise LisaException(
                "The kernel config CONFIG_KEXEC or CONFIG_KEXEC_FILE is not set. "
                "Kdump is not supported."
            )

Reply:
@LiliDeng Updated.